.gitlab-ci.yml

```diff
@@ -292,3 +301,31 @@ publish-docs:
     - git push origin gh-pages --force
   after_script:
     - rm -rf .git/ ./*
+
+#### stage:                        fuzz
+
+fuzz-tests:
+    stage:                           fuzz
+    <<:                              *docker-env
+    variables:
+        # The QUICKCHECK_TESTS default is 100
+        QUICKCHECK_TESTS:            1000
```

Comment:
What does this stand for? Seconds? Number of test runs?

Reply:
"Number of tests to run" from the docs.

Reply:
and what's the default amount?

Reply:
Default amount is 100.